cpp/lib/IAuth.cpp

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2024 Snowflake Computing, Inc. All rights reserved.
+ */
+
+#include <string>
+#include "snowflake/Exceptions.hpp"
```

Comment:
remove this to ensure no exception involved.

Reply:
I removed unnecessary codes related to the exception.